app/src/main/java/io/gnosis/safe/ui/transactions/TransactionListFragment.kt

```diff
@@ -119,9 +115,21 @@ class TransactionListFragment : SafeOverviewBaseFragment<FragmentTransactionList
 
     override fun onResume() {
         super.onResume()
+        binding.transactions.scrollToPosition(0)
```

Comment:
Why are we doing this? I would only scroll to top if we have new data ... it is a major inconvenience that the android app scrolls up even if nothing in the list changed. Did you ever try browsing a long transaction list?

Reply:
I agree. But AFAICT it is specified that way. We should change the specification as it is really annoying with longer tx lists.

Reply:
yes, we could discuss it after release and then change this behaviour for both android and ios.

Reply:
iOS only scrolls up if they really load data ... they don't do a preemptive scrolling up when nothing changed ... this is bad ux and just so that it feels the same we should not introduce bad ux ....

Reply:
Also could you reference where this is specified please.